### PR TITLE
Bug 1225881 - Fix failure in test_clock_alarm_in_lockscreen.py

### PR DESCRIPTION
**Pull request by mwargers**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=36f3db7a927af578e105ff72c11482d2b0ec4bcf

File: tests/python/gaia-ui-tests/gaiatest/tests/functional/clock/test_clock_alarm_in_lockscreen.py [hunk 1]
```diff
@@ -38,7 +39,8 @@ def test_clock_set_alarm_in_lockscreen(self):
                 return new Date(yr, mth, day, 1, 0, 0).getTime();""")
         self.data_layer.set_time(_seconds_since_epoch)
 
-        new_alarm = self.clock.tap_new_alarm()
+        alarm = Alarm(self.marionette)
```
Comment (JohanLorenzo) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/clock/test_clock_alarm_in_lockscreen.py]: I'd user `self.clock.switch_view('alarm')` so the user flow is a bit clearer

Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/clock/test_clock_alarm_in_lockscreen.py]: Ok, I didn't know about that, I'll change it.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=052be08b970caba2d559c383734a2fa768f00233

**Pull request merged**